gasstation/cache.go

```diff
@@ -0,0 +1,14 @@
+package gasstation
+
+import "math/big"
+
+type blockFee struct {
+	baseFee      *big.Int
+	gasUsedRatio float64
+	blobBaseFee  *big.Int
+	blobGasRatio float64
+}
+
+type blockPercents struct {
+	ascEffectivePriorityFees []*big.Int
+}
```

Comment:
i think can directly add into `gasstation.go`? seems too less of code change to make a new file